lib/marqeta/card.rb

```diff
@@ -19,10 +19,14 @@ def retrieve_pan
       self.attributes_hash = ApiCaller.new("cards/#{token}/showpan").get
     end
 
+    def create_client_access
+      ClientAccess.api_create(card_token: token)
+    end
+
     private
 
     def accessible_attributes
-      super + %i[state pan card_product_token]
+      super + %i[state pan card_product_token fulfillment_status]
```

Comment:
@sughandj  where is fulfillment_status used?

Reply:
@teresa-lo The `fulfillment_status` tells us if physical card order is "SHIPPED", "DELIVERED", etc. Will be used in Frontend eventually.

Reply:
I think it would be nice if we could store all of the different statuses as constants here and then expose them as methods as needed. So for example `card.shipped?`, `card.delivered?`. This way the user of the gem doesn't need to read the Marqeta documentation in order to use the `fulfillment_status`.